src/libsystemd/sd-device/device-monitor.c

```diff
@@ -212,6 +216,10 @@ int device_monitor_new_full(sd_device_monitor **ret, MonitorNetlinkGroup group,
                 }
         }
 
+        r = uid_range_load_userns(&m->uid_range, &m->n_uid_range, NULL);
+        if (r < 0)
+                log_monitor_errno(m, r, "Failed to load UID range, ignoring: %m");
```

Comment:
it's highly unlikely we'll need this. please initialize this lazily in check_sender_uid(), so that we only do this if the uid == 0 check doesn't suffice